### DISCUSSION TITLE
Moving local project to Heroku

### DISCUSSION THREAD

**Discussion by Nemzytch** [strapi/strapi#8251] [Legacy Topics]
**Short version** : How do you migrate a big strapi local project to any online environnemnt to allow online data add via the admin. And what's the best option. 

Hello we've been working on a project with strapi locally, we've built different collection types, with components in it ( a lot ) and a lot of different entries in every of those collection types and components.

We didnt think about how we would move this online, to allow our clients to add entries themselves ( only create entries, not create any new collection type etc. ) . So, we are wondering what would be the best process to migrate what we have locally , as we cant really reupload everything one by one because there is actually way too much entries and it wouldnt be viable.

And what would be the less painful way to do it : heroku ? Digital ocean ? Else ?

Thank a lot in advance.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
